### PR TITLE
Clients per mesh

### PR DESCRIPTION
**Pull request by Moorviper**
This counts all clients in a local mesh

demo:
https://meshviewer.fulda.freifunk.net/#!v:m;n:6872514ae045

### CODE REVIEW

**Comment by plumpudding** [comment]
I will have to take a closer look at this, but it could take a while. I'm quite busy at the time with the upcoming exams and other hobbies.

**Comment by viisauksena** [comment]
als feedback und dank:
ich hab die Änderungen bei mir "cherrypicked" und funktioniert prächtig ! :) merci merci 
http://openfreiburg.de/freifunk/meshviewer/#!v:m;n:f4f26d3fe762
https://cccfr.de/git/viisauksena/meshviewer2/commit/1cbfa236bfcc2da5ffd645c85996c88ac1d924b7

File: lib/infobox/node.js [hunk 1]
```diff
@@ -133,14 +133,70 @@ define(["moment", "numeral", "tablesort", "tablesort.numeric"],
     if (!d.flags.online)
       return undefined
 
+    var meshclients = getMeshClients(d)
+    resetMeshClients(d)
+    var before = "     ( "
```
Comment (plumpudding) [inline lib/infobox/node.js]: why these spaces after and before the open- / close bracket?

File: lib/infobox/node.js [hunk 2]
```diff
@@ -133,14 +133,70 @@ define(["moment", "numeral", "tablesort", "tablesort.numeric"],
...
+    })
+
+    return
+  }
+
+  function showMeshClients(d) {
+    if (!d.flags.online)
+      return undefined
+
+  var meshclients = getMeshClients(d)
```
Comment (plumpudding) [inline lib/infobox/node.js]: indention....

**Pull request merged**